Firestore/core/src/firebase/firestore/util/status.h

```diff
@@ -161,6 +198,10 @@ inline Status::State::State(const State& s)
                                                    : s.platform_error->Copy()) {
 }
 
+inline Status::State::State(Error code, std::string msg)
```

Comment:
Nit: can you make `msg` an rvalue reference (`std::string&& msg`)? `Status` is used widely enough that it _might_ be worthwhile for performance (avoids one extra move).

Reply:
Done.